src/Util/Datastore.php

```diff
@@ -67,14 +67,16 @@ public function getFile(string $filePath): array
     /**
      * Moves an uploaded file to datastore disk location.
      *
-     * @param array $fileStream File stream resource object.
+     * @param array  $fileStream      File stream resource object.
+     * @param string $destinationPath File destination path on datastore.
      *
      * @return string
      */
-    public function addFile(array $fileStream): string
+    public function addFile(array $fileStream, string $destinationPath): string
```

Comment:
I would just call this filePathName. This should be only the file path name of the file, and not the dictated file location/destination of the file. Of course destinationPath was already used in the code below. I'm just making this comment because it could be confusing.